### ISSUE TITLE
Wishlist: combobox for selecting brand in models edit (when admin)

### ISSUE THREAD

**Issue by rmescandon** [CanonicalLtd/serial-vault#110]
In case logged in user is Admin, maybe a drop down list with all possible values could be shown when adding/editing a model instead of an input text for the Brand field. There is a limited brands group for every tenant.

**Comment by slimjim777** [comment]
Yes, this sounds good. Also, when brand is selected, the signing-keys drop-downs should be filtered so the user only sees the keys for the same account.
